tvb/datatypes/projections.py

```diff
@@ -78,7 +124,17 @@ class ProjectionRegionEEG(framework.ProjectionRegionEEGFramework,
         
     
     """
-    pass
+    @staticmethod
+    def from_file(source_file='projection_EEG_region.npy', instance=None):
```

Comment:
well I like it more than hdf5 because it does no impose the user to have the right name for the array.
But we can use hdf5, so what should be the name of the field? 
gain? projection_matrix? leadfield?